src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.cs

```diff
@@ -16,7 +16,18 @@ namespace System.Net.Http.Tests
     public class HttpClientHandlerTest
     {
         readonly ITestOutputHelper _output;
+        private const string Username = "testuser";
```

Comment:
@kapilash You don't need to move these 2 constants.  They are test constants and not server constants since httpbin.org allows dynamic username/password to be sent/used as part of the uri path itself.